docs/governance.rst

```diff
@@ -0,0 +1,128 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
+maintainer of ``mybinder.org``, a public service and demonstration of
+the BinderHub technology.
+
+Membership
+~~~~~~~~~~
+
+Membership in the Binder Project is open to anyone who is nominated by
```

Comment:
There should be some mention of inclusion. Nomination by existing members tends to reinforce the status quo unless conscious outreach is done. Outreachy is a good start but more should be done.

Reply:
👍  do you have a suggestion for a sentence we can use here?

For me this document is more like a legal reference document so right now it is very brief on philosophy/intentions :-/

My best idea right now is to take https://github.com/jupyterhub/team-compass/issues/42#issuecomment-451458207 and turn it into an issue to get the ball rolling. Then insert a link to that issue here and then replace it with a link to the outcome of the issue once we have it. Suggested sentence: "To find out more about the process and how to get started see link-to-issue."

Reply:
While it could get a mention in the heading here, I think it definitely warrants an explicit point in the "responsibilities" below.

Reply:
I like the idea of explicitly making inclusion and increasing the diversity of the community part of the responsibilities of being a "red" team member (or just a member in general)